src/main/java/hello/HelloController.java

```diff
@@ -4,10 +4,23 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 
 @RestController
-public class HelloController {
-    
+public class HelloController implements Comparable {
+    public static String foo = "";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this "public static foo" field final [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube:9000/coding_rules#rule_key=squid%3AS1444)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make foo a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube:9000/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)